src/Components/EventCard.js

```diff
@@ -32,8 +35,10 @@ const EventCard = (props) => {
                 return Images.events.sports;    
             case "THEATRE":
                 return Images.events.theatre;
-            default:
+            case "":
                 return Images.events.coffee;
+            default:
+                return lowerType;
```

Comment:
Why change the default? Couldn't `lowerType` be nothing in this case which would render a broken image?